apps/meteor/server/models/raw/Roles.ts

```diff
@@ -179,6 +179,14 @@ export class RolesRaw extends BaseRaw<IRole> implements IRolesModel {
 		return this.find(query, options || {});
 	}
 
+	findCustomRoles(options?: FindOptions<IRole>): FindCursor<IRole> {
+		const query: Filter<IRole> = {
+			protected: false,
```

Comment:
I see there's usually no huge amount of data stored in this collection, but I still wonder if we should add an index here. wdyt @sampaiodiego ?
Other than that, this looks fine to me

Reply:
This shouldn't affect the QA, I'm approving this one and we can go back on this one for the external review

Reply:
we should add one, but it's not a blocking requirement for this PR